.changes/v3.13.0/1246-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Fix a missing Edge Gateway filter bug in `vcd_nsxt_alb_edgegateway_service_engine_group` resource
+  (issue #1245) [GH-1246]
```

Comment:
Should we add the link to the issue? (As far as I know, it will not be added by the changelog scripts).

Reply:
Added.